cmd/serverless/main.go

```diff
@@ -73,7 +73,7 @@ func main() {
 		os.Exit(-1)
 	}
 
-	// handle SIGTERM
+	// handle SIGTERM signal
```

Comment:
small change to trigger integration tests

Reply:
Not sure if it applies here but usually when I need to commit something to trigger the CI I run a 'git commit --allow-empty' to push an empty commit.

Reply:
Yes, but this does not apply there. 
We trigger integration on serverless only if specific files are touch ( see : https://github.com/DataDog/datadog-agent/blob/main/.github/workflows/serverless-integration.yml#L5 )
I think this is about the change and we will run integration tests on all commits in a near future